packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -18,35 +26,19 @@ const WorkflowStepStore = types
       return []
     },
 
-    isThereANextStep () {
-      const nextStep = self.getNextStepKey()
-      return nextStep && nextStep !== 'summary'
-    },
-
-    isThereAPreviousStep () {
-      const validStepReference = isValidReference(() => self.active)
-      if (validStepReference) {
-        const firstStep = self.steps.keys().next()
-        return self.active.stepKey !== 'summary' && self.active.stepKey !== firstStep.value
-      }
-
-      return false
-    },
-
     get isThereTaskHelp () {
       const tasks = self.activeStepTasks
 
       return tasks.some(task => task.help)
     },
 
     get shouldWeShowDoneAndTalkButton () {
-      const isThereANextStep = self.isThereANextStep()
       const workflow = tryReference(() => getRoot(self).workflows?.active)
       const classification = tryReference(() => getRoot(self).classifications?.active)
 
```

Comment:
Let's check for a valid step reference. Used below.
```suggestion
const step = tryReference(() => self.active)
```